### PR TITLE
Feature facebook improvement

### PR DESCRIPTION
**Pull request by khangaridb**

### CODE REVIEW

**Comment by shinebayar-g** [comment]
test comment

File: src/trackers/facebook.js [hunk 1]
```diff
@@ -100,6 +100,220 @@ export class SaveWebhookResponse {
     }
   }
 
+  /*
+   * Get page access token
+   */
+  async getPageAccessToken() {
```
Comment (batamar) [inline src/trackers/facebook.js]: Remove async, await

File: src/db/models/Customers.js [hunk 2]
```diff
@@ -86,6 +86,10 @@ const facebookSchema = mongoose.Schema(
       optional: true,
       label: 'Facebook photo',
     }),
+    profileLink: field({
```
Comment (batamar) [inline src/db/models/Customers.js]: remove this fields

File: src/trackers/facebook.js [hunk 3]
```diff
@@ -316,12 +544,14 @@ export class SaveWebhookResponse {
...
-  /*
+  /**
    * Get or create customer using facebook data
    * @param {String} fbUserId - Facebook user id
+   * @param {Object} kind - Facebook data type messenger or status
+   *
    * @return Previous or newly created customer object
    */
-  async getOrCreateCustomer(fbUserId) {
+  async getOrCreateCustomer(fbUserId, kind) {
```
Comment (batamar) [inline src/trackers/facebook.js]: Remove kind

File: src/trackers/facebook.js [hunk 4]
```diff
@@ -191,13 +423,19 @@ export class SaveWebhookResponse {
     let messageText = value.message;
```
Comment (batamar) [inline src/trackers/facebook.js]: Remove messageText checks

**Pull request merged**